tsdb/config.go

```diff
@@ -98,8 +100,14 @@ type Config struct {
 }
 
 func NewConfig() Config {
+	defaultEngine := DefaultEngine
+	if engine := os.Getenv("INFLUXDB_DATA_ENGINE"); engine != "" {
```

Comment:
I am adding this message for now, so it's crystal clear in the CircleCI output what is happening.